.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint",
+	       "cypress"
```

Comment:
The idea is that each item of the plugins array starts at the same column, or has the same indentation spaces. In this case "cypress" item have mixed tab and spaces, please use only spaces to indent, that would fix the issue :)